examples/02_investigating_dirty_categories.py

```diff
@@ -46,24 +52,23 @@
 # To simplify understanding, we will focus on the column describing the
 # employee's position title:
 
-values = data[['employee_position_title', 'gender']]
-values.insert(0, 'current_annual_salary', employee_salaries.y)
+values = data[["employee_position_title", "gender"]]
+values.insert(0, "current_annual_salary", employee_salaries.y)
 
 #########################################################################
 # String similarity between entries
-# -------------------------------------------------
+# ---------------------------------
 #
 # That's where our encoders get into play.
-# In order to robustly embed dirty semantic data, the SimilarityEncoder
+# In order to robustly embed dirty semantic data, the :class:`SimilarityEncoder`
 # creates a similarity matrix based on the 3-gram structure of the data.
```

Comment:
Maybe
```suggestion
# creates a similarity matrix based on a n-gram representation of the data.
```
It might be worth mentioning (perhaps in a "note" callout) that by default, for optimal results, it uses a range of different n-grams. Or perhaps that's a detail. I'm not sure.

Reply:
I think this is too detailed. If they are interested they should see the encoder docs.

Reply:
Fair enough :)